README.md

```diff
@@ -61,15 +66,23 @@ For building the driver `docker` and `GO` should be installed on the system
 
    Create an image pull secret in your cluster
 
-   1. Review and retrieve the following values for your image pull secret.
+   1. ibmcloud login to the target region
+
+   2. Run - ibmcloud cr region-set global
+
+   3. Run - ibmcloud cr login
+
+   4. Run - ibmcloud ks cluster config --cluster \<cluster-name\> admin
```

Comment:
Can you update this to
`ibmcloud ks cluster config --cluster \<cluster-name\> --admin`

Reply:
> One case which I didnt test, was what was the maximum length of UID/GID that the user can give. If there is a limit on that, we can add a check. Just a corner case.
> 
> Otherwise the PR looks good to me. Thanks

This error will come from VPC RIAAZ if there is any issue with gid and uid. Lets think later if we need to add more validations at driver side